css/jumpstart-lab.css

```diff
@@ -2,3 +2,7 @@
   float: right;
   margin-top: 6px;
 }
+
+body.header-bkg-text-light #block-search-form input.search-query {
+  background-color: #fff !important;
```

Comment:
I made the selector more specific and removed the important. it was applying the white, but after expanding, it was going transparent. I believe some javascript was doing some work there.